.env.sample

```diff
@@ -0,0 +1,2 @@
+MYSQL_DATABASE=
+MYSQL_ROOT_PASSWORD=
```

Comment:
Tutaj jest podobnie jak we frontendzie i tamtym plikiem .env. Wystarczy skopiować sobie .env.sample i usunąć .sample. Później wpisać odpowiednią nazwę bazy i hasło w .env.